src/ncp/ncp_base.cpp

```diff
@@ -1091,6 +1092,7 @@ void NcpBase::LinkRawReceiveDone(otRadioFrame *aFrame, otError aError)
     SuccessOrExit(error = OutboundFrameSend());
 
 exit:
+    OT_UNUSED_VARIABLE(error);
```

Comment:
Logging errors sounds reasonable to me. As for blacklistEntry, it already seems to print some information in case of error, so I'll remove this variable.